services/account/index.ts

```diff
@@ -1,14 +1,35 @@
-import { Application } from '@loopback/core';
-import { AccountController } from './controllers/AccountController';
+import {Application, ApplicationConfig} from '@loopback/core';
+import {RestBindings, RestComponent, RestServer} from '@loopback/rest';
+import {AccountRepository} from './repositories/account/index';
+import {juggler, DataSourceConstructor} from '@loopback/repository';
+import {AccountController} from './controllers/AccountController';
 
 class AccountMicroservice extends Application {
   private _startTime: Date;
 
-  constructor() {
-    super();
-    const app = this;
-    app.bind('http.port').to(3001);
-    app.controller(AccountController);
+    constructor(options?: ApplicationConfig) {
+        options = Object.assign(
+            {},
+            {
+                components: [RestComponent],
+                rest: {
+                    port: 3001
+                }
+            },
+            options
+        );
+        super(options);
+
+        const dataSource: juggler.DataSource = new DataSourceConstructor('local-fs', {
```

Comment:
I think it would make more sense to define `datasource` elsewhere in its own file (for e.g. `/datasources/mem.datasource.ts`), and only bind it in this file.